### PR TITLE
Feature/vmo 5604 add component names as classes

### PR DESCRIPTION
**Pull request by laserpants**

### CODE REVIEW

File: src/components/interaction-designer/Block.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <div @click="selectBlock">
+  <div @click="selectBlock" class="block">
```
[the remark this answers is not in the document]
Comment (laserpants) [reply src/components/interaction-designer/Block.vue:2]: That seems like a good rule. I thought that `BaseBlock` would have been a sensible name, but 

1. there is already one component named `base-block`, and
2. this doesn't seem to fit with the definition of a _Base Component_ from the Vue docs:

> Base components (a.k.a. presentational, dumb, or pure components) that apply app-specific styling and conventions should all begin with a specific prefix, such as `Base`, `App`, or `V.`

Maybe we could call it `InteractionBlock` (`interaction-block`). 

@safydy-r @object-required What do you think?

[the remark this answers is not in the document]
Comment (laserpants) [reply src/components/interaction-designer/Block.vue:2]: Good point. I say we consider this an exception to the rule then, as you suggested. @object-required, what do you think, please?

**Comment by yeboaasare** [comment]
Hi @laserpants I get the crash below when I ran yarn serve
`yarn run v1.22.18
warning package.json: "dependencies" has dependency "node-sass" with range "^4" that collides with a dependency in "devDependencies" of the same name with version "^4.14.1"
error @floip/flow-builder@0.4.0: The engine "node" is incompatible with this module. Expected version "^14.19.1". Got "14.16.1"
error Commands cannot run with an incompatible environment.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.`

NB: my yarn version is 1.22.18

cc/ @safydy-r

**Comment by laserpants** [comment]
@yeboaasare What version of `node` are you using? We currently support `v14`, which was made explicit in the `package.json` file, with the following field added recently:

```
  "engines": {
    "node": "^14.19.1"
  },
```

**Review by safydy-r — APPROVED** [review]

**Comment by bulatgab** [comment]
@yeboaasare The error you had will be fixed in the [next PR](https://github.com/FLOIP/flow-builder/pull/235)

**Pull request merged**